lib/prerequisites.sh

```diff
@@ -3,14 +3,16 @@
 source lib/functions.sh
 
 detectCompose
+configDirectory="/etc/bridgehead/"
+componentDirectory="/srv/docker/bridgehead/"
```

Comment:
Is the convention to use camel case variables for local variables? Otherwise, please convert to upper case

Reply:
Thx for checking, added your suggestions